beesdoo_website_shift/static/src/js/script.js

```diff
@@ -0,0 +1,25 @@
+odoo.define("beesdoo_website_shift.script", function (require) {
```

Comment:
@MatteoLdx Un small comment here to explain what does this javascript code would be wonderfull. :)